sources/packages/backend/apps/api/src/route-controllers/student/_tests_/e2e/student.institutions.utils.ts

```diff
@@ -0,0 +1,42 @@
+import {
+  createFakeInstitutionLocation,
+  saveFakeApplication,
+  saveFakeStudent,
+} from "@sims/test-utils";
+import {
+  InstitutionTokenTypes,
+  getAuthRelatedEntities,
+} from "../../../../testHelpers";
+import { DataSource } from "typeorm";
+import { Application, Student } from "@sims/sims-db";
+
+/**
+ * Save student application for College C
+ * with a College C location.
+ * @param dataSource data source to connect the database.
+ * @returns student and application details with application location.
+ */
+export async function saveStudentApplicationForCollegeC(
+  dataSource: DataSource,
+): Promise<{ student: Student; collegeCApplication: Application }> {
+  const student = await saveFakeStudent(dataSource);
+  const { institution } = await getAuthRelatedEntities(
+    dataSource,
+    InstitutionTokenTypes.CollegeCUser,
+  );
+  const collegeCLocation = createFakeInstitutionLocation(institution);
+  const application = await saveFakeApplication(dataSource, {
+    student,
+    institutionLocation: collegeCLocation,
+  });
+  return {
+    student,
+    collegeCApplication: application,
+  };
+}
+
+export const INSTITUTION_BC_PUBLIC_ERROR_MESSAGE =
```

Comment:
I see the value of creating these as const but please consider that other PR will be using it also. Should we maybe move those to the API `testHelpers/auth`?